Cargo.toml

```diff
@@ -159,12 +159,12 @@ fnv = "1.0.7"
 sha2 = "0.10.8"
 
 # Serde
-serde = { version = "1.0.199", features = ["derive"] }
+serde = { version = "1", features = ["derive"] }
 serde_dynamo = "4.2.14"
-serde_json = { version = "1.0.116", features = ["preserve_order"] }
+serde_json = { version = "1", features = ["preserve_order"] }
```

Comment:
This wasn't a fixed version. `=1.0.116` would have been one. `1.0.116` means 1.0.116 or anything newer within the same major version. So, `cargo update` would have sufficed here.

Reply:
ah my bad then, misunderstood it. should I revert the Cargo.toml then?